src/sql/base/common/errors.ts

```diff
@@ -3,16 +3,10 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// localizable strings
-
-export const InvalidProvider = 'Provider is invalid';
-
-/**
- * Feature names
- */
-export const RestoreFeatureName = 'restore';
-export const BackupFeatureName = 'backup';
-
-export const MssqlProviderId = 'MSSQL';
-export const notebookModeId = 'notebook';
-
+export function invalidProvider(name?: string): Error {
+	if (name) {
+		return new Error(`Invalid provider: ${name}`);
```

Comment:
>`Invalid provider: ${name}` [](start = 19, length = 27)

Do you know if these are displayed to the user anywhere? If so they should be localized

Reply:
I didn't see anywhere where we were specifically looking for this error and showing it. Generally we should avoid showing errors to the user since they could be coming from anywhere.